beta-src/src/state/interfaces/SavedOrders.ts

```diff
@@ -16,7 +16,7 @@ export interface SupportMoveChoice {
 
 interface SharedMeta {
   allowedBorderCrossings?: TerritoryClass[];
-  convoyToChoices?: TerritoryClass[];
+  convoyToChoices?: string[]; //TerritoryClass[];
```

Comment:
I should also take a pass at some point to add | null to the field that I observed could be null, and fix any fallout for that. If you know of any other type errors in our interfaces we should fix them.